libraries/ms-helper/inc/thermistor.h

```diff
@@ -27,9 +27,8 @@ typedef struct ThermistorStorage {
 StatusCode thermistor_init(ThermistorStorage *storage, GPIOAddress thermistor_gpio,
                            ThermistorPosition position);
 
-// Fetch the temperature reading in milliCelsius from the MCU's ADC
-StatusCode thermistor_get_temp(ThermistorStorage *storage, uint16_t *temperature_millicelcius);
+// Fetch the temperature reading in decicelsius from the MCU's ADC
+StatusCode thermistor_get_temp(ThermistorStorage *storage, uint16_t *deci_celsius);
```

Comment:
Could probably use `temp_decicelsius/temp_dc` or something instead?

Reply:
yah
working on it rn